teraserver/python/modules/FlaskModule/API/user/UserQueryProjectAccess.py

```diff
@@ -208,8 +208,9 @@ def post(self):
                 # Check if we must remove access for that site
                 if json_project['project_access_role'] == '':
                     # No more access to that project for that user group - remove all access!
-                    TeraServiceAccess.delete_service_access_for_user_group_for_project(
-                        id_user_group=json_project['id_user_group'], id_project=json_project['id_project'])
+                    TeraServiceAccess.delete_service_access_for_user_group(id_service=Globals.opentera_service_id,
```

Comment:
In that case, yes - this is the specific API for TeraServer project access. There's a separate generic ServiceAccess API that could be used if not dealing with TeraServer. In the future, it might indeed be useful to combine both (in fact, three of them, as the Site Access has a similar structure).